packages/web/src/pages/chat-page/components/ReactionPopupMenu.tsx

```diff
@@ -0,0 +1,53 @@
+import { ComponentType, MutableRefObject } from 'react'
+
+import { reactionOrder, ReactionTypes } from '@audius/common'
+import { Popup, PopupPosition } from '@audius/stems'
+
+import {
+  reactionMap,
+  ReactionProps
+} from 'components/notification/Notification/components/Reaction'
+
+import styles from './ReactionPopupMenu.module.css'
+
+const reactionList: [ReactionTypes, ComponentType<ReactionProps>][] =
+  reactionOrder.map((r) => [r, reactionMap[r]])
+
+export const ReactionPopupMenu = ({
```

Comment:
Can't reuse this from tiping notifs?

Reply:
Reused what I could. I don't think that one is a popup